src/App.tsx

```diff
@@ -1,204 +1,75 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { FilterTypes } from './types/FilterTypes';
+import { getPreparedTodos } from './utils/GetPrepatedTodos';
+import { getAmountOfActiveTodos } from './utils/getAmountOfActiveTodos';
+import { Footer } from './components/Footer';
+import { Errors } from './utils/Errors';
+import { ErrorsField } from './components/ErrorsField';
+import { Header } from './components/Header';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterInstructions, setFilterInstructions] = useState<FilterTypes>(
+    FilterTypes.All,
+  );
```

Comment:
I think it looks better, what do u think?
```suggestion
  const [
    filterInstructions, 
    setFilterInstructions
  ] = useState<FilterTypes>(FilterTypes.All);
```